l10n_it_fatturapa_in/readme/CONTRIBUTORS.rst

```diff
@@ -3,3 +3,5 @@
 * Alessio Gerace
 * Sergio Zanchetta <https://github.com/primes2h>
 * Giovanni Serra <giovanni@gslab.it>
+* Gianmarco Conte <gconte@dinamicheaziendali.it>
+* Giuseppe Borruso <gborruso@dinamicheaziendali.ite>
```

Comment:
Non è un po' eccessivo per una modifica così semplice? Al massimo direi un contributors in più comunque.

Reply:
ok ora lo tolgo